bower.json

```diff
@@ -1,12 +1,14 @@
 {
```

Comment:
u dont hav to worry / change this file manually, at least for now. Try to keep them as the original, when the build process is in place, these will be taken care for you.